client/models/user.coffee

```diff
@@ -5,3 +5,6 @@ sd = require('sharify').data
 module.exports = class User extends Backbone.Model
 
   urlRoot: "#{sd.API_URL}/users"
+
+  @isEditorialTeam: (user) ->
```

Comment:
I'm cool with this, but it feels like it would make more sense to add this as an instance method and use `@get('type') is 'Admin'....` and `user.isEditorialTeam()`. Not sure if that's reasonably feasible in the current setup.